apps/condo/domains/common/components/BaseSearchInput/index.tsx

```diff
@@ -45,6 +46,7 @@ export const BaseSearchInput = <S extends string>(props: ISearchInput<S>) => {
         style,
         infinityScroll,
         value,
+        searchAgainDependencies = [],
```

Comment:
it looks too complex to understand what's going on. why is it not possible to use re-render the component when changing props?